--- a/data/etc/sysactions.conf.in
+++ b/data/etc/sysactions.conf.in
@@ -43,10 +43,18 @@
 # etc.
 #
 # user and group name can use glob matches (* == all for example) like the
 # shell. as can action names allowed or denied.

-action:   halt      @HALT@
-action:   reboot    @REBOOT@
-action:   suspend   @SUSPEND@
-action:   hibernate @HIBERNATE@
+# Use runit commands for Void Linux
+# poweroff and reboot via runit, suspend/hibernate via zzz
+action:   halt      /sbin/poweroff
+action:   reboot    /sbin/reboot
+action:   suspend   /usr/bin/zzz
+action:   hibernate /usr/bin/ZZZ
 action:   /bin/mount /bin/mount
 action:   /bin/umount /bin/umount
 action:   /usr/bin/eject /usr/bin/eject
@@ -57,14 +65,23 @@

 # root is allowed to do anything - but it needs to be here explicitly anyway
 user:     root      allow: *
 # members of operator, staff and admin groups should be able to do all
 group:    operator  allow: *
 group:    staff     allow: *
 group:    admin     allow: *
 group:    sys       allow: *
 group:    wheel     allow: *
 group:    adm       allow: *
 # common "user" groups for "console users" on desktops/laptops
 group:    dialout   allow: *
 group:    disk      allow: *
 group:    adm       allow: *
 group:    cdrom     allow: *
 group:    floppy    allow: *
 group:    audio     allow: *
 group:    dip       allow: *
 group:    plugdev   allow: *
 group:    netdev    allow: *
 group:    bluetooth allow: *
 group:    video     allow: *
 group:    voice     allow: *
 group:    fax       allow: *
 group:    tty       allow: *
 # put in a list of other users and groups here that are allowed or denied etc.
 # e.g.
 # user:     myuser     allow:  *
 # user:     another    allow:  suspend hibernate
 #
 #
 # uncomment this line to enable eeze mounting for users
 # user:   someuser allow: /bin/mount /bin/umount /usr/bin/eject
 #
 #
 # deny everyone else by default
 user:     *         deny:  *
